build-image/first-boot.bash

```diff
@@ -149,29 +151,29 @@ if hash python3 2>/dev/null; then bash /boot/webif.bash reinsure_running; fi
 echo -n "$(timestamp) [openHABian] Installing git package... "
 if apt-get -y install git &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
 
-echo -n "$(timestamp) [openHABian] Cloning myself from ${REPOSITORYURL}, ${CLONEBRANCH} branch... "
 if [ -d /opt/openhabian/ ]; then cd /opt && rm -rf /opt/openhabian/; fi
-if git clone -q -b "$CLONEBRANCH" "$REPOSITORYURL" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+# shellcheck disable=SC2154
+echo -n "$(timestamp) [openHABian] Cloning myself from ${repositoryurl}, ${clonebranch} branch... "
+if git clone -q -b "$clonebranch" "$repositoryurl" /opt/openhabian; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
```

Comment:
actually it seems we have broken all 3 developer options in build.bash: local-test, inject_build_repo.
local-test has two sed commands which need to be adapted (target: remove L#154, L#157, and at best also L#156)
inject_build_repo: last sed command to be adapted, and file is no longer $1 but /etc/openhabian.conf

Reply:
not sure what you mean here. Marked file is first-boot.bash but you talk about build.bash,
and L154 is no sed on neither of these ?
Please recheck against latest code I've just uploaded

Reply:
yes, I am talking about build.bash.
build.bash is patching first-boot.bash with a few sed commands. Those relay on the exact wording in first-boot.bash. So we need to adapt those commands in build.bash L152/L153.
local-test removes the got clone command with sed, we need to adapt to our new text.
inject_build_repo patches the cit clone url, this needs to be applied to /etc/openhabian.conf, so build.bash L63 needs this change.
Not a big thing....

Reply:
ok I think I understand but as you're more familiar with that would you please patch build.bash?
After we merged #683 
On a quick look, build.bash L152 still clones master so that should be reworked, too

Reply:
fine for me, it is anyway broken right now

Reply:
I quickly replaced those occurrences in local-test but are you sure about replacing $1 by "/etc/openhabian.conf" in last sed line of inject_build_repo ? sed looks to replace "master http://github.com....." but this is not contained as a string in there.
Wouldn't it rather be to replace the repositoryurl= and clonebranch= parameters ?